__PySiddhiProxy/threadfix_c_code/README.MD

```diff
@@ -0,0 +1,6 @@
+threadfix is used in Python 3.4+ to treat a threading related bug introduced in Python 3.4.
```

Comment:
Add license header

Reply:
Since this is a markdown file, do we still need to add the license header?